src/view/abstract_tela_cadastro.py

```diff
@@ -1,21 +1,22 @@
+from abc import ABC
 from typing import List
 
 from src.utils.enum_operacoes import Operacao
 from src.utils.enum_tipo_cadastro import TipoCadastro
 from src.view.abstract_tela import AbstractTela
 
 
-class AbstractTelaCadastro(AbstractTela):
+class AbstractTelaCadastro(AbstractTela, ABC):
```

Comment:
```suggestion
class AbstractTelaCadastro(AbstractTela):
```

Esse ABC é realmente necessário?